New table naming strategy makes the tests fail on the sample application

I just merged #1799 and I only tested the application with MySQL to see the new table names, and that was a huge mistake!!

So it makes the `mvn test` fail on the sample application https://github.com/jhipster/jhipster-sample-app

The error message is:

    Caused by: org.hibernate.HibernateException: Missing column: bankAccount_id in SAMPLEAPPLICATION.PUBLIC.OPERATION

@deepu105 I'm assigning this to you are you are the author of the PR, of course.

There is also one slight formatting issue: there are two lines after the "id" field.

Ok ill take look I was doing a direct test may be i missed to add strategy
in test .yml
On 2 Sep 2015 01:33, "Julien Dubois" <notifications@github.com> wrote:

> I just merged #1799
> <https://github.com/jhipster/generator-jhipster/pull/1799> and I only
> tested the application with MySQL to see the new table names, and that was
> a huge mistake!!
>
> So it makes the mvn test fail on the sample application
> https://github.com/jhipster/jhipster-sample-app
>
> The error message is:
>
> Caused by: org.hibernate.HibernateException: Missing column: bankAccount_id in SAMPLEAPPLICATION.PUBLIC.OPERATION
>
> @deepu105 <https://github.com/deepu105> I'm assigning this to you are you
> are the author of the PR, of course.
>
> There is also one slight formatting issue: there are two lines after the
> "id" field.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/issues/1962>.
>

--- a/app/templates/src/test/resources/config/_application.yml
+++ b/app/templates/src/test/resources/config/_application.yml
@@ -31,7 +31,7 @@ spring:<% if (databaseType == 'sql') { %>
         generate-ddl: false
         hibernate:
             ddl-auto: none
-            naming-strategy: org.hibernate.cfg.EJB3NamingStrategy
+            naming-strategy: org.springframework.boot.orm.jpa.hibernate.SpringNamingStrategy
         properties:
             hibernate.cache.use_second_level_cache: false
             hibernate.cache.use_query_cache: false

